src/services/completions.ts

```diff
@@ -10,7 +12,17 @@ namespace ts.Completions {
         AutoImportSuggestions = "6",
         JavascriptIdentifiers = "7"

```

Comment:
Need `DeprecatedLocalDeclarationPriority ` and other deprecated enum values here,